snapcraft.yaml

```diff
@@ -559,8 +559,6 @@ parts:
     source-depth: 1
     source-type: git
     plugin: make
-    build-environment:
```

Comment:
No it was needed to do shallow clone to avoid 2gb clone https://github.com/canonical/lxd-pkg-snap/commit/5c9aee86cb162393daa8a382d8f3ec2826c38425

Reply:
The build process tried to get git tag but wasnt available in shallow clone  so had to set it manually

Reply:
That's right, sorry for not build testing this before submitting. My previous commit broke the build :/
The PR builds fine now.